src/shogun/lib/parameter_observers/ParameterObserverCV.cpp

```diff
@@ -177,7 +164,7 @@ CrossValidationStorage* CParameterObserverCV::get_observation(int run) const
 	return obs;
 }
 
-const int CParameterObserverCV::get_num_observations() const
+const size_t CParameterObserverCV::get_num_observations() const
```

Comment:
Please use the shogun int32_t to avoid sign warnings (need to explicitly cast and take care or overflows)